modules/admin_manual/pages/configuration/server/occ_commands/core_commands/_file_commands.adoc

```diff
@@ -120,6 +122,32 @@ For example, if the user's id was "john" and the `--path` value was "tree/apple"
 
 `files:cleanup` tidies up the server's file cache by deleting all file entries that have no matching entries in the storage table.
 
+== The files:remove-storage command
+
+When a mount point gets removed, the storage ID and file cache related info stay in the database. This can over time consume considerable amount of database space respectively can slow down database queries. Withe the `files:remove-storage` command, these orphans can be removed. As a rule of thumb though not necessary in any case, using this command should be considered to run in single user mode to avoid any conflicting access.
+
+[source,bash,subs="attributes+"]
+----
+{occ-command-example-prefix} files:remove-storage --help
+ Usage:
+   files:remove-storage [options] [--] [<storage-id>]...
+----
+
+=== Arguments
+
+[width="100%",cols="23%,70%",]
+|===
+| `storage-id` | Remove a storage from the storages table and related files from the filecache table.
```

Comment:
seems a wrong description for the argument

Reply:
Fixed, thanks for catching 👍